sklearn/feature_selection/from_model.py

```diff
@@ -18,6 +82,8 @@ class _LearntSelectorMixin(TransformerMixin):
     ``feature_importances_`` or ``coef_`` attribute to evaluate the relative
     importance of individual features for feature selection.
     """
+    @deprecated('Support to use estimators as feature selectors will be '
```

Comment:
Could you please check that no examples or narrative doc triggers this warning? If they do, they should be changed to use `SelectFromModel`.